src/client.ts

```diff
@@ -78,6 +80,20 @@ const replaceProfile = (options: ICourierClientConfiguration) => {
   };
 };
 
+const patchProfile = (options: ICourierClientConfiguration) => {
```

Comment:
I think for new items we want it to be `client.profiles.patch(...)`. Trying to bring a little order to the code.